recipe/meta.yaml

```diff
@@ -12,60 +12,31 @@ source:
     sha256: {{ sha256 }}
 
 build:
-    number: 1000
-    script: python setup.py install --single-version-externally-managed --record record.txt
+    number: 0
+    script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   build:
     - python
-    - setuptools
+    - pip
   run:
     - python
+    - freetype
+    - future
 
 test:
-  imports:
-    - pyglet
-    - pyglet.app
-    - pyglet.canvas
-    - pyglet.extlibs
-    - pyglet.libs
-    - pyglet.libs.x11  # [linux]
-    - pyglet.libs.darwin  # [osx]
-    - pyglet.libs.darwin.cocoapy  # [osx]
-    - pyglet.libs.win32  # [win]
-
-    # Skip all of the audio driver imports for now...
-
-    # skip to avoid installing audio drivers on Windows, see
-    # https://www.openal.org/platforms/
-    #- pyglet.media.drivers.openal  # [not win or linux]
-
-    # Again, skip the driver-specific import
-    #- pyglet.media.drivers.pulse  # [not win]
-
-    # Skip the carbon test because carbon isn't present on recent versions
-    # of OSX.
-    # - pyglet.window.carbon  # [osx]
-
-    - pyglet.window.win32  # [win]
-
-    # Would be nice to enable these eventually on non-Windows:
-    - pyglet.font  # [win]
-    - pyglet.gl  # [win]
-    - pyglet.graphics  # [win]
-    - pyglet.image  # [win]
-    - pyglet.image.codecs  # [win]
-    - pyglet.input  # [win]
-    - pyglet.media  # [win]
-    - pyglet.media.drivers  # [win]
-    - pyglet.media.drivers.directsound  # [win]
-    - pyglet.window  # [win]
-    - pyglet.text  # [win]
-    - pyglet.text.formats  # [win]
```

Comment:
Why are we removing the import tests?

Reply:
@jakirkham 
Some tests require access to a (virtual) screen on Linux; therefore, the tests need to be run through `xvfb-run` on that platform. To do that, I created a Python script, `run_tests.py`, that does all the `import` testing. It is run directly (`python run_tests.py`) on Windows and macOS, and via `xvfb-run` on Linux. I have added a comment explaining this in 6e1dc47

An alternative could be to only list the (few) tests that actually require a virtual screen in the test script, and keeping the rest of the `import tests` in `meta.yaml`. But that would put the tests in two different places, and could be confusing?